README.md

```diff
@@ -51,6 +51,17 @@ const StyledComponentWithMediaQuery = styled.Text`
     font-size: 14px;
   }
 `;
+
```

Comment:
we should mention container queries more prominently in the readme somewhere and provide a link to the spec